app/scripts/templates/test_template.mustache

```diff
@@ -37,10 +37,12 @@
     <div class="input-row password-row">
       <input type="password" class="password" id="password" pattern=".{8,}" required />
       <input id="show-password" type="checkbox" class="show-password" aria-controls="password" data-novalue />
+      <div class="input-help input-help-focused input-help-signup">{{#t}}Must be at least 8 characters{{/t}}</div>
     </div>
     <div class="input-row password-row">
       <input type="password" class="password" id="vpassword" pattern=".{8,}" required />
       <input id="show-password" type="checkbox" class="show-password" aria-controls="vpassword" data-novalue />
+       <div class="input-help input-help-complete-password">{{#t}}Must be at least 8 characters{{/t}}</div>
```

Comment:
you don't need the extra single space in front of the `<div`, this gives it extra indentation. see above on line 40 for correct formatting that you added.